### ISSUE TITLE
Many scripts don't work with galaxy.yml

### ISSUE THREAD

**Issue by nsoranzo** [galaxyproject/galaxy#5733]
For example:
```
(.venv)[galaxy]$ python scripts/cleanup_datasets/cleanup_datasets.py config/galaxy.yml -d 10 -1
Traceback (most recent call last):
  File "scripts/cleanup_datasets/cleanup_datasets.py", line 548, in <module>
    main()
  File "scripts/cleanup_datasets/cleanup_datasets.py", line 101, in main
    config_parser.read(ini_file)
  File "/usr/lib64/python2.7/ConfigParser.py", line 305, in read
    self._read(fp, filename)
  File "/usr/lib64/python2.7/ConfigParser.py", line 512, in _read
    raise MissingSectionHeaderError(fpname, lineno, line)
ConfigParser.MissingSectionHeaderError: File contains no section headers.
file: config/galaxy.yml, line: 1
'uwsgi:\n'
```

**Comment by martenson** [comment]
some of these are more important than others and the `cleanup_datasets.py` from your gist is one of the most important I think :/

**Comment by nsoranzo** [comment]
I suppose we could use `load_app_properties()` from `lib/galaxy/util/properties.py`

**Comment by jmchilton** [comment]
We should be using load_app_properties everywhere - without it environment variables also aren't read so things like the docker image do not work correctly. I'll see if I can take a look.

**Comment by nsoranzo** [comment]
A simplified version of what can be used from `lib/galaxy/util/script.py` is:
```python
from galaxy.util.properties import find_config_file, load_app_properties


def app_properties(config_file=None, app='galaxy', config_section=None):
    if not config_file:
        config_file = find_config_file(app)
    return load_app_properties(config_file=config_file, config_section=config_section)
```

**Comment by nsoranzo** [comment]
Fixed by #5769.
